openedx/core/djangoapps/coursegraph/tasks.py

```diff
@@ -388,21 +387,12 @@ def authenticate_and_create_graph(credentials):
     neo4j_user = credentials['user']
     neo4j_password = credentials['password']
 
-    authenticate(
-        "{host}:{port}".format(
-            host=host, port=https_port if secure else http_port
-        ),
-        neo4j_user,
-        neo4j_password,
-    )
-
     graph = Graph(
-        bolt=True,
+        protocol='http',
         password=neo4j_password,
         user=neo4j_user,
-        https_port=https_port,
-        http_port=http_port,
-        host=host,
+        address=host,
+        port=https_port if secure else http_port,
```

Comment:
I could have changed the management command to just accept a single `port`, but this was quicker.

Reply:
This shouldn't slow down the run of the management command, right? It seems from my googling that the bolt driver is what makes this go fast (note that in my own testing on your branch, nothing seemed slow, but it was also only migrating the two devstack default courses).

Reply:
@connorhaugh Gah, I have no idea how it'll affect performance. Even just in terms of data correctness, it's probably a good idea to not switch the protocol as part of this upgrade. Thanks for pointing this out -- I'd switched to HTTP to get it working at first, but forgot to put it back to Bolt.

I'll push a change to switch back to Bolt. The py2neo API has changed a bit (it no longer needs the HTTP ports), so I'll need to change the management command arguments a bit too.